assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/workers/workers.tf

```diff
@@ -52,6 +52,65 @@ resource "aws_autoscaling_group" "workers" {
   ])
 }
 
+# Worker IAM Policy
+resource "aws_iam_instance_profile" "worker-instance-profile" {
+  name = "${var.name}-instance-profile"
```

Comment:
This doesn't seem to work. Should it be `var.pool_name`? Or `${var.cluster_name}-${var.pool_name}-instance-profile` although this will be too long so maybe `${var.cluster_name}-${var.pool_name}-iprof`.